docs/items/assessments.md

```diff
@@ -0,0 +1,25 @@
+---
+id: assessments
+title: Compliance Assessments in CAT 
+sidebar_position: 3
+---
+
+## Compliance Assessments in CAT 
+<p>In CAT, a Compliance Assessment is considered as the process of determining to what extent a service, object, organisation, or capabilities comply with a set of criteria, based on reproducible tests. <br/>Validated users can contribute external assessments or contribute a self-assessment. </p>
+
+#### Verification of a Compliance Assessment
+<p>Compliance assessment is considered successful if it complies with a minimum set of standards.</p>
+<p>These standards (Principles) differ depending on the motivation that guides the necessity of the assessment, upon the role of the user in the ecosystem. </p>
+Principles contain a set of MANDATORY criteria that need to be to be satisfied, in order to decide it's success as well as a set of OPTIONAL criteria, that support further the compliance of the assessment.
+<p>While submitting the compliance assessment/self-assessment the user needs to answer to tests and provide proofs, in order to achieve the success of the criterio.</p>
+<p>Guidelines and best practices are provided to support each test, and help the user to decide actions in order to comply with the required standards.</p>
+
+
+#### Compliance Assessment Visibility
+<p>A Compliance Assessment can be private (user-owned) or public and upon this depends it's visibility through the CAT Service.</p>
```

Comment:
A public assessment is an assessment published to zenodo. This is a feature that will be implemented in the near futuer.